airesources/Scala/project/Dependencies.scala

```diff
@@ -0,0 +1,5 @@
+import sbt._
+
+object Dependencies {
+  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.0.3"
```

Comment:
Is this actually used?

Reply:
You're right, it's not used. I removed it.